### PR TITLE
Update PyCharm test setup docs for pytest

### PR DESCRIPTION
**Pull request by jmbowman**
This seems to work for me, but I'm not a heavy user of running tests through PyCharm; let me know if this looks incorrect or incomplete to anyone.

### CODE REVIEW

**Review by clintonb — APPROVED** [review]
It's probably worth noting that, if you attempt to run a single test class that subclasses `TestCase` (by right-clicking on the class), PyCharm will not use `py.test`, but will default to using `unittest`.

**Comment by jmbowman** [comment]
After tinkering around with it for a while and hunting through the PyCharm bug tracker, I discovered that the Django support feature currently overrides the default test runner.  So you need to either disable Django support or (starting in 2017.3 EAP) opt not to use the Django test runner.  I've updated the doc accordingly.

File: docs/pycharm_integration.rst [hunk 1]
```diff
@@ -197,34 +197,34 @@ configuration with the following options:
...
-5. Python Interpreter: Choose the Docker Compose interpreter for this
-   service.
+Then make two changes in PyCharm's preferences:
 
-6. Working directory: /edx/app/edxapp/edx-platform
+1. Set "Tools -> Python Integrated Tools -> Default test runner" to "py.test".
 
-7. Path mappings (add mapping):
+2. In "Languages & Frameworks -> Django", uncheck "Enable Django Support".
+   Starting in PyCharm 2017.3 (the EAP of which is already available), this
```
Comment (clintonb) [inline docs/pycharm_integration.rst:220]: We should have a ticket to update these docs once 2017.3 is released.

**Review by clintonb — APPROVED** [review]

File: docs/pycharm_integration.rst (hunk 1 above)
Comment (jmbowman) [reply docs/pycharm_integration.rst:220]: Done: [PLAT-1787](https://openedx.atlassian.net/browse/PLAT-1787)

**Review by macdiesel — APPROVED** [review]
Tested and works in my environment.

**Pull request merged**